site/content/contribute/getting-started/code-review.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Code Review"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* Code changes must be reviewed by at least two [core committers](/contribute/getting-started/core-committers/#core-committers).
+* Documentation changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+  * Product managers may ask for reviews from [core committers](/contribute/getting-started/core-committers/#core-committers) and [QA testers](/contribute/getting-started/core-committers/#qa-testers) as required.
+* Any change may be verified by a [QA tester](/contribute/getting-started/core-committers/#qa-testers) as required.
+
+If you are a community member seeking a review
+----------------------------------------------
+
+1. Submit your pull request.
+    * Follow the [contribution checklist](https://developers.mattermost.com/contribute/getting-started/contribution-checklist/).
+2. Reach out to the author of the ticket your PR resolves.
```

Comment:
Typically Eric's GitHub account is the author of all the help wanted issues so I'm not sure if this works unless they look at the JIRA ticket as well. Not really sure of a foolproof way for them to know who to mention, unless they've been interacting with a core committer somewhere about the ticket already

Reply:
Agreed. I've removed the mention of the author, and updated the language here.